website-next/pages/index.js

```diff
@@ -82,89 +56,99 @@ function HeroSection() {
+        <ImageContainer />
+        <ImageContainer />
+        <ImageContainer />
+        <ImageContainer />
+        <ImageContainer />
+        <ImageContainer />
       </div>
-      <div className='px-2 pt-6 sm:pt-16 lg:pt-32 max-w-screen-2xl mx-auto'>
-        <GhostBrowser className='w-full object-cover'>
-          <HostedImage src='/screenshots/screenshot2.png' />
-        </GhostBrowser>
-      </div>
-    </>
-  )
-}
-
-function AlwaysLiveSection() {
-  return (
-    <>
-      <div className='max-w-7xl mx-auto pt-40 text-center'>
-        <MainTitle>Always Live</MainTitle>
+      <div className='text-right w-2/4 pl-4'>
+        <MainTitle>Immediately Familiar</MainTitle>
         <Subtitle>
-          See code and design change instantly. Instantly jump between between design and preview
-          mode. See and edit the same component in multiple configurations.
+          Utopia includes a though-out design tool to inspect, edit and create.
+          It uses your code as a source of truth, and you can see the changes it makes to it in real time.
         </Subtitle>
       </div>
-      <div className='px-2 pt-6 sm:pt-16 lg:pt-32 max-w-screen-2xl mx-auto'>
-        <GhostBrowser className='w-full object-cover'>
-          <HostedImage src='/screenshots/screenshot2.png' />
-        </GhostBrowser>
-      </div>
-    </>
-  )
-}
+    </div>
+  </div>
+)
+
+const LandingPageSection4 = props => (
+  <div className='pt-24 pb-24' style={{width: '100%', backgroundColor: '#181818'}}>
+    <div className='max-w-7xl mx-auto text-center'>
+      <MainTitle dark>Designed to Code</MainTitle>
+      <Subtitle dark>
+        Utopia includes a browser-based development environment for React. It’s powered by
+        Microsoft’s VSCode, takes seconds to spin up, and give you a real-time preview of your app
+        and components. And it includes everything you expect:
+      </Subtitle>
+    </div>
+    <div className='max-w-7xl pt-16 mx-auto'>
+      <LandingScrollableSection />
+    </div>
+  </div>
+)
 
-export default function LandingPage() {
+const OpenSourceSection = props => (
+  <div className='pt-24 pb-24' style={{width: '100%', backgroundColor: '#FFFFFF'}}>
+    <div className='max-w-7xl mx-auto text-center'>
+      <MainTitle>Proudly Open Source</MainTitle>
+      <Subtitle>
+        Utopia is an open-source project under the MIT license.
+      </Subtitle>
+    </div>
+  </div>
+)
+
+// TODO this is the landing page, it should be the default export once it's enabled
+// export default function LandingPage() {
+function LandingPage() {
   return (
     <div>
       <Header />
       <div
-        className='relative bg-blue-100 overflow-hidden bg-no-repeat bg-cover'
-        style={{ backgroundImage: 'url(/backgrounds/mesh-pinkblue-sky.jpg)' }}
+        // className='relative bg-blue-100 overflow-hidden bg-no-repeat bg-cover'
+        // style={{ backgroundImage: 'url(/backgrounds/mesh-pinkblue-sky.jpg)' }}
```

Comment:
```suggestion
```

Delete me? I'm not sure if these were kept deliberately so that we can re-enable them